src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -1051,10 +1056,13 @@ namespace WaterThermalTanks {
         int HeatingSourceNum;    // Index to DX cooling coil (heat source for desuperheater)
         int NumAlphas;           // Number of elements in the alpha array
         int NumNums;             // Number of elements in the numeric array
+        int WAHPHeatingSourceNum; // Index to WA cooling coil (heat source for desuperheater)
+
         // unused1208  INTEGER                     :: NumArgs                 ! Number of elements in the object (alpha + numeric)
         int RackNum;                              // Index to refrigrated display case rack
         int CondNum;                              // Index to refrigration condenser
         int DXCoilNum;                            // Index to DX coils
+        int WAHP_EQ_Coil_Num;                     // Index to WaterToAir Heat Pump equation fit cooling coils
```

Comment:
Again, this is just a local counter used in the loops that are searching yet again for the source.